### ISSUE TITLE
tedlium/s5_r2/local/nnet3/tuning/run_lstm_1a.sh needs to be updated in order to use xconfig_to_configs.py

### ISSUE THREAD

**Issue by Potomac** [kaldi-asr/kaldi#1787]
Hello,

I notice that the default lstm script in tedlium/s5_r2/local/nnet3/tuning/run_lstm_1a.sh has not been updated to the new config method, 

if we run this script with the last git version of kaldi then we will have this error message :

You have num_hidden_layers=3 (real meaning: your config files 
                        are intended to do discriminative pretraining).  Since Kaldi 5.2, 
                        "this is no longer supported --> use newer config-creation scripts, 
                        "i.e. xconfig_to_configs.py.".format(field_value))

as a kaldi beginner I don't know how to fix this error, I assume that the problematic lines are these : 

if [ $stage -le 12 ]; then
  echo "$0: creating neural net configs"
  config_extra_opts=()
  [ ! -z "$lstm_delay" ] && config_extra_opts+=(--lstm-delay "$lstm_delay")
  steps/nnet3/lstm/make_configs.py  "${config_extra_opts[@]}" \
    --feat-dir $train_data_dir \
    --ivector-dir $train_ivector_dir \
    --ali-dir $ali_dir \
    --num-lstm-layers $num_lstm_layers \
    --splice-indexes "$splice_indexes " \
    --cell-dim $cell_dim \
    --hidden-dim $hidden_dim \
    --recurrent-projection-dim $recurrent_projection_dim \
    --non-recurrent-projection-dim $non_recurrent_projection_dim \
    --label-delay $label_delay \
    --self-repair-scale-nonlinearity 0.00001 \
  $dir/configs || exit 1;
fi

can anyone can post a quick workaround ?

thanks

**Comment by danpovey** [comment]
The same issue exists in the ASPIRE recipe.
@hhadian, do you have time to try creating a fix?  You don't have to test it.  I'll look over the config files to see if I think they're equivalent.

BTW, I think the final "0" corresponds to the presoftmax layers and the first set of splice indexes (e.g. -1,0,1 or -2,-1,0,1) corresponds to the LDA layer.

**Comment by hhadian** [comment]
Will do it

**Comment by danpovey** [comment]
resolved in #1790
